src/RageSoundReader_Extend.cpp

```diff
@@ -155,7 +155,7 @@ bool RageSoundReader_Extend::SetProperty( const RString &sProperty, float fValue
 {
 	if( sProperty == "StartSecond" )
 	{
-		m_iStartFrames = std::lrint( fValue * this->GetSampleRate() );
+		m_iStartFrames = static_cast<int>( fValue * this->GetSampleRate() );

```

Comment:
What's the rationale for keeping `fValue` as a float and `m_iStartFrames` as an int now? Above the source/dest frames are `int64_t`s.

I'm trying to understand why some frames are ints and others are int64_ts.

Reply:
No reason to keep them int / float, I just didn't want to change too much code without testing myself to make sure it doesn't break.  Happy to make these 64-bit as well.